DevelopersGuide.adoc

```diff
@@ -725,7 +725,7 @@ You can start the compiler server with:
 
 [source,bash]
 ----
-$ npx shadow-cljs server
+$ npx shadow-cljs watch main
 ----
 
 and navigate to the URL it prints out for the compiler server (usually http://localhost:9630).
```

Comment:
It says *very* clearly in the text right here that you will need to use the build menu.  In watch mode, turning OFF the build in the UI kills the entire build tool, which is annoying.  I prefer the instruction the way it is, but I'll add a warning to call attention.

Reply:
That makes sense. Thanks!